test/internal/e2e/kubebuildertest.go

```diff
@@ -136,14 +178,21 @@ func (kt *KubebuilderTest) CleanUp() error {
 	return nil
 }
 
-func (kt *KubebuilderTest) runCommand(cmd *exec.Cmd) error {
+// RunKubectlCommand is a general func to run kubectl commands
+func (kt *KubebuilderTest) RunKubectlCommand(cmdOptions []string) (string, error) {
+	cmd := exec.Command("kubectl", cmdOptions...)
+	output, err := kt.runCommand(cmd)
+	return string(output), err
+}
+
+func (kt *KubebuilderTest) runCommand(cmd *exec.Cmd) ([]byte, error) {
 	cmd.Dir = kt.Dir
 	cmd.Env = os.Environ()
 	command := strings.Join(cmd.Args, " ")
 	output, err := cmd.Output()
 	if err != nil {
-		return fmt.Errorf("%s failed with error: %s", command, string(output))
+		return output, fmt.Errorf("%s failed with error: %v", command, err)
```

Comment:
Capturing output as part of the error is super useful during the debugging because the error returned by cmd.Output() is generally of the form "Exit 255: ......" and doesnt' contain much info.

Reply:
sounds reasonable